### PR TITLE
fix: dokan redirection issue

### PR DESCRIPTION
**Pull request by xaviranik**

### CODE REVIEW

File: includes/Dokan.php [hunk 1]
```diff
@@ -51,10 +51,12 @@ public function manager_permission( $valid ) {
      * @return void
      */
     public function frontend_permissions( $valid ) {
-        if ( dokan_is_user_seller( dokan_get_current_user_id() ) ) {
-            if ( dokan_is_seller_enabled( dokan_get_current_user_id() ) ) {
+        if ( dokan_is_user_seller( get_current_user_id() ) ) {
```
Comment (kapilpaul) [inline includes/Dokan.php:54]: can we make these two conditions in one line?

Comment (xaviranik) [reply includes/Dokan.php:54]: Yes we can.

Comment (kapilpaul) [reply includes/Dokan.php:54]: Please update.

**Comment by tareq1988** [comment]
@xaviranik we can not release without merging this PR, right?

**Comment by xaviranik** [comment]
> @xaviranik we can not release without merging this PR, right?

Yes bhai.

File: includes/Dokan.php (hunk 1 above)
Comment (xaviranik) [reply includes/Dokan.php]: updated

**Pull request merged**